homework/english/skiba_olga/lecture2/grammar2.md

```diff
@@ -0,0 +1,38 @@
+# Упражнения Past Simple и Present Perfect
+
+### Упражнение 1.  Put the verbs in brackets in the Past Simple or in the Present Perfect.
+
+1. I've just had a nice pot of coffee. Would you like a cup?
+2. I've not seen Steve this morning yet.
+3. Carol and I are old friends. I've known her since I was a freshman in high school.
+4. Maria has had a lot of problems since she come to this country.
```

Comment:
... since she cAme to this country

Reply:
Thank you. I'm so inconsiderate. made all the changes